schema/policy.go

```diff
@@ -364,6 +373,10 @@ func (auth *AdminAuthorization) IsAdmin() bool {
 	return true
 }
 
+func (auth *AdminAuthorization) IsDomainAdmin() bool {
+	return true
```

Comment:
what do you think - should be true or false? in both cases we can distinguish admin from domain admin

Reply:
Doesn't this type represent admin scoped to admin?

Reply:
you mean `AdminAuthorization`? it does

Reply:
Then if think it should be `false`, since it's not a **domain** admin

Reply:
Ok, i just followed the logic that admin is a superset of domain admin so that's why i set it to true. Guys, could you vote with thumbs up if i should change it to `false` and thumbs down otherwise?

Reply:
Then it's better to call this function `HasDomainAdminPrivileges`. I agree with @jkmar.

Reply:
done in different manner